app.py

```diff
@@ -57,7 +57,11 @@ def update_movie(movie_id: int):
     return redirect(f'/movies/{movie_id}')
 
 
-@app.post('/movies/<int:movie_id>/delete')
+@app.route('/movies/<int:movie_id>/delete', methods=['POST'])
 def delete_movie(movie_id: int):
-    # TODO: Feature 6
-    pass
```

Comment:
I'm not really sure you need the pass here. It might interfere with potential errors.